README.reproducible.md

```diff
@@ -0,0 +1,167 @@
+that Gitian uses to build in, but gitian-tools also supports LXC.
+
+The following steps will allow you to build Armory reproducibly using Gitian.
+
+1. Obtain gitian-tools from [Devrandom's GitHub][Devrandom's GitHub].
+2. Run `bin/make-base-vm --suite trusty`. Armory uses Ubuntu Trusty as the OS
+   of the VM to build in. Add `--lxc` if using LXC.
+3. If using LXC, run `export USE_LXC=1`.
+4. Run `bin/gbuild --commit BitcoinArmory=autotools-gitian ../BitcoinArmory/gitian-descriptors/bitcoin-armory-osx.yml`. The commit option is optional and
+   specifies a Git tag, branch name, or commit hash. By default it is master.
+   The command ends with the path to the appropriate YAML file in the Armory
+   source tree. In this case, the OS X Gitian descriptor is used to indicate
+   that we want to build for OS X.
+5. TODO: Include instructions about signing and verifying.
+
+Note that the OS X build requires you have the Mac SDK. See
+[fetch and build inputs][fetch and build inputs].
+
+Note that you may (read will) need to grab the dependency sources manually. See
+[seed cache][seed cache]. Just replace bitcoin in the instructions with
+BitcoinArmory or whatever your Armory source tree is named.
+
+##Linux
+
+###Reproducible
+
+For Linux reproducible builds, we have used the
+[Debian Reproducible Build Toolchain][Debian Toolchain]. We have found the
+toolchain to be better suited to producing reproducible Debian packages simply
+due to the fact that it was designed specifically with Debian packages in mind.
+
+For more on producing a reproducible Debian package of Armory, read
+[dpkgfiles/make\_deb\_package.README](dpkgfiles/make_deb_package.README)
+
+###Non-reproducible
+
+If you don't want to build for Linux reproducibly, or you are building for a
+system that does not support deb or rpm packages, you can just build using the
+Autotools-based system directly. This has only been tested when building on
+Linux.
+
+    cd path-to-armory-source
+    ./autogen.sh
+    ./configure # don't need to specify host if building on same architecture as host
+    make
+    make install # optional
+
+Specify `--host x86_64-linux-gnu` or `--host i686-linux-gnu` if you are
+building for 64-bit from a 32-bit machine or building 32-bit from a 64-bit
+machine respectively.
+
+If you don't run `make install`, you can just run `python ArmoryQt.py` from the
+Armory source tree. If you run `make install`, you should have `armory` on your
+path. So you can just run `armory`.
+
+###Fedora Package (RPM)
+
+RPM packages are not officially supported. They are a convenience to users.
+Read [rpmfiles/armory.spec.README](rpmfiles/armory.spec.README) for more info.
+
+##OS X
+
+###Reproducible
+
+The OS X reproducible build system uses Gitian. Refer to the Gitian
+instructions and use gitian-descriptors/bitcoin-armory-osx.yml as the Gitian
+descriptor.
+
+###Non-reproducible
+
+You can use the Autotools-based build system to build Armory.app. This has been
```

Comment:
This should probably be set aside for now, maybe under "FUTURE DIRECTIONS (DO NOT USE)" or something similar. AFAIK, Windows and OS X are still going to use the old build systems until the Py3/Qt5 upgrade. So, for this, people will need to run "python build_app.py" from osxbuild. (I'll probably have to upgrade the script once this is pushed. I'll take care of that.)